corehq/apps/accounting/models.py

```diff
@@ -4076,6 +4078,10 @@ class Meta(object):
         abstract = True
 
 
+class AccountCommunicationHistory(CommunicationHistoryBase):
```

Comment:
Do all of `CommunicationType.CHOICES` apply to each of the subclasses of  `CommunicationHistoryBase`? Like would `UPCOMING_MONTHLY_PAYMENT` ever be linked to an `Invoice` or `CustomerInvoice`?

Reply:
likely not, but we are also moving towards offloading the invoice generation and comms to stripe, so these models will not be relevant once that is done